### ISSUE TITLE
Java 8 Distributable

### ISSUE THREAD

**Issue by sualeh** [spannm/ucanaccess#3]
Please maintain the Java 8 distributable. There are still many organizations that use Java 8 in production.
(see [Welcome to (the new home of) UCanAccess](https://sourceforge.net/p/ucanaccess/discussion/general/thread/f43cb9b7fb))

**Comment by spannm** [comment]
I have moved to Java 11 as the minimum required Java version to take advantage of new features, optimizations, and security enhancements introduced in Java 11 LTS, which will ultimately contribute to a more robust and maintainable codebase.

Maintaining support for multiple target run-time Java versions in parallel is unfortunately out of scope for me.

**Comment by manuelvgr** [comment]
[heart]         Manuel Vicente Garcia Rodriguez reacted to your message:
________________________________

**Comment by manuelvgr** [comment]
[0]     Manuel Vicente Garcia Rodriguez reacted to your message:
________________________________
